schemas/aerleon-definitions.schema.json

```diff
@@ -34,6 +34,11 @@
       "description": "Specifies an IP address or CIDR IP address range expression.",
       "type": "string"
     },
+    "fqdn": {
+      "description": "Specifies a fully qualified domain name with two or more labels.",
+      "type": "string",
+      "pattern": "^(?!:\/\/)(?=.{1,255}$)((.{1,63}.){1,127}(?![0-9]*$)[a-z0-9-]+.?)$"
```

Comment:
`(.{1,63}\.)`
I assume the escape on the period isn't necessary for this one?

Reply:
Also same for the other escaped period from where this was copied from.

Reply:
Fixed, good catch